app/src/main/kotlin/co/netguru/baby/monitor/client/feature/voiceAnalysis/WavFileGenerator.kt

```diff
@@ -27,26 +27,64 @@ object WavFileGenerator {
         sampleRate: Int,
         byteRate: Int
     ) = Single.fromCallable {
-        checkAvailableSpace(context)
+        checkAvailableSpace(
+            context
+        )
         val formatter = DateTimeFormatter.ofPattern(DATE_PATTERN)
         val file = File(
             context.getDir(DIRECTORY_NAME, Context.MODE_PRIVATE),
             "crying_${LocalDateTime.now().format(formatter)}.wav"
         )
 
         DataOutputStream(FileOutputStream(file)).use { output ->
-            writeString(output, "RIFF") // chunk id
-            writeInt(output, 36 + rawData.size) // chunk size
-            writeString(output, "WAVE") // format
-            writeString(output, "fmt ") // subchunk 1 id
-            writeInt(output, 16) // subchunk 1 size
-            writeShort(output, 1.toShort()) // audio format (1 = PCM)
-            writeShort(output, 1.toShort()) // number of channels
-            writeInt(output, sampleRate) // sample rate
-            writeInt(output, byteRate) // byte rate
-            writeShort(output, 2.toShort()) // block align
-            writeShort(output, 16.toShort()) // bits per sample
-            writeString(output, "data") // subchunk 2 id
+            writeString(
+                output,
+                "RIFF"
+            ) // chunk id
+            writeInt(
+                output,
+                36 + rawData.size
+            ) // chunk size
+            writeString(
+                output,
+                "WAVE"
+            ) // format
+            writeString(
+                output,
+                "fmt "
+            ) // subchunk 1 id
+            writeInt(
+                output,
+                16
+            ) // subchunk 1 size
+            writeShort(
+                output,
+                1.toShort()
+            ) // audio format (1 = PCM)
+            writeShort(
+                output,
+                1.toShort()
+            ) // number of channels
+            writeInt(
+                output,
+                sampleRate
+            ) // sample rate
+            writeInt(
+                output,
+                byteRate
+            ) // byte rate
+            writeShort(
+                output,
+                2.toShort()
+            ) // block align
+            writeShort(
+                output,
+                16.toShort()
+            ) // bits per sample
+            writeString(
+                output,
+                "data"
+            ) // subchunk 2 id
```

Comment:
Out of curiosity , why such formatting?

Reply:
oh must've gone through while doing auto formatting. Not intended.